website/pages/en/firehose/_meta.js

```diff
@@ -74,7 +74,6 @@ export default () =>
       items: {
         'design-principles': 'Design Principles',
         'firehose-starter': 'Firehose Acme',
-        'new-blockchains': 'New Blockchains',
         'why-integrate-the-firehose': 'Why Integrate the Firehose?',
```

Comment:
```suggestion
        'benefits': 'Why Integrate the Firehose',
```
Not sure why this page was renamed because [the `h1` is still "Why Integrate the Firehose"](https://github.com/streamingfast/firehose-docs/blob/master/integrate-new-chains/benefits.md#why-integrate-the-firehose), but it _was_, so.